css/css-flexbox-1/negative-margins-001.html

```diff
@@ -34,7 +32,7 @@
     }
 </style>
 <body>
-    <p>You should see a green rectangle with a black border, 60px wide. You should see no red.</p>
+    <p>You should see a green rectangle with a black border, 40px wide. You should see no red.</p>
```

Comment:
Really? Interesting, on Windows - FF/Chrome/Edge all render the green to a total of 40px so I adjusted it to match "reality" so that you don't see any red.

![image](https://user-images.githubusercontent.com/865244/27662514-f4647264-5c12-11e7-833c-978552e86c49.png)